CONTRIBUTING.md

```diff
@@ -9,15 +9,15 @@ If you encounter an issue with the project, you are welcome to submit a
 Before that, please search for similar issues. It's possible that someone has already reported the problem.
 
 # PRs
-Much of the code in this repository is generated from
-Cloudant's OpenAPI specification. This means PRs
-to code, tests and sometimes even markdown
-are often inappropriate and we may need to make
-changes to the specification instead. For this reason
-we ask that an **issue is always opened prior to a PR**
-to give us the opportunity to discuss the best place
-for the change before investing your effort on a patch
-that we may not be able to accept.
+We ask that an **issue is always opened prior to a PR** to give us the
+opportunity to discuss the best place for the change before investing
+your effort on a patch that we may not be able to accept.
+
+The code in the repository is partly generated from the Cloudant OpenAPI
+specification. This means PRs to code, tests and sometimes even markdown
+are often inappropriate and we may need to make changes to the specification
+instead. Please, first [look at the list of files](.github/CODEOWNERS) that
+_should not_ be modified.
```

Comment:
We should remove `partly` as it opens the door for questions like "which parts?"
I've also suggested some other changes:
```suggestion
This repository is generated from the Cloudant OpenAPI
specification. PRs to source code, tests and markdown documentation
might be improper if the change is required within the OpenAPI specification.
Please [review the list of files](.github/CODEOWNERS) that _should not_ be modified.
```

Reply:
For me, it is clear the `partly` refers to the list of files mentioned in one of the later sentences. That is why I would keep the first sentence as-is, mainly because not the whole repository, but a part of it is generated. However, I fancy the simplification of the second sentence.

What do you think about this:
```suggestion
The code in the repository is partly generated from the Cloudant OpenAPI
specification. PRs to source code, tests, and markdown
might be improper if the change is required within the OpenAPI specification. 
Please, first [look at the list of files](.github/CODEOWNERS) that
_should not_ be modified.
```

Reply:
No need to change, keep as-is.